server/src/test/java/io/druid/server/coordinator/DruidCoordinatorBalancerTest.java

```diff
@@ -395,4 +400,269 @@ public void testRun2() throws IOException
+    EasyMock.expect(druidServer1.getMaxSize()).andReturn(100L).atLeastOnce();
+    EasyMock.expect(druidServer1.getSegments()).andReturn(segments).anyTimes();
+    EasyMock.expect(druidServer1.getSegment(EasyMock.<String>anyObject())).andReturn(null).anyTimes();
+    EasyMock.replay(druidServer1);
+
+    EasyMock.expect(druidServer2.getName()).andReturn("2").atLeastOnce();
+    EasyMock.expect(druidServer2.getTier()).andReturn("normal").anyTimes();
+    EasyMock.expect(druidServer2.getCurrSize()).andReturn(0L).atLeastOnce();
+    EasyMock.expect(druidServer2.getMaxSize()).andReturn(100L).atLeastOnce();
+    EasyMock.expect(druidServer2.getSegments()).andReturn(new HashMap<String, DataSegment>()).anyTimes();
+    EasyMock.expect(druidServer2.getSegment(EasyMock.<String>anyObject())).andReturn(null).anyTimes();
+    EasyMock.replay(druidServer2);
+
+    EasyMock.expect(druidServer3.getName()).andReturn("3").atLeastOnce();
+    EasyMock.expect(druidServer3.getTier()).andReturn("extra").anyTimes();
+    EasyMock.expect(druidServer3.getCurrSize()).andReturn(0L).atLeastOnce();
+    EasyMock.expect(druidServer3.getMaxSize()).andReturn(100L).atLeastOnce();
+    EasyMock.expect(druidServer3.getSegments()).andReturn(segments).anyTimes();
+    EasyMock.expect(druidServer3.getSegment(EasyMock.<String>anyObject())).andReturn(null).anyTimes();
+    EasyMock.replay(druidServer3);
+
+    EasyMock.expect(druidServer4.getName()).andReturn("4").atLeastOnce();
+    EasyMock.expect(druidServer4.getTier()).andReturn("extra").anyTimes();
+    EasyMock.expect(druidServer4.getCurrSize()).andReturn(0L).atLeastOnce();
+    EasyMock.expect(druidServer4.getMaxSize()).andReturn(100L).atLeastOnce();
+    EasyMock.expect(druidServer4.getSegments()).andReturn(new HashMap<String, DataSegment>()).anyTimes();
+    EasyMock.expect(druidServer4.getSegment(EasyMock.<String>anyObject())).andReturn(null).anyTimes();
+    EasyMock.replay(druidServer4);
+
+    // Mock stuff that the coordinator needs
+    coordinator.moveSegment(
+        EasyMock.<ImmutableDruidServer>anyObject(),
+        EasyMock.<ImmutableDruidServer>anyObject(),
+        EasyMock.<String>anyObject(),
+        EasyMock.<LoadPeonCallback>anyObject()
+    );
+    EasyMock.expectLastCall().anyTimes();
+    EasyMock.replay(coordinator);
+
+    LoadQueuePeonTester peon1 = new LoadQueuePeonTester();
+    LoadQueuePeonTester peon2 = new LoadQueuePeonTester();
+    LoadQueuePeonTester peon3 = new LoadQueuePeonTester();
+    LoadQueuePeonTester peon4 = new LoadQueuePeonTester();
+
+    ListeningExecutorService exec = MoreExecutors.listeningDecorator(
+        Executors.newFixedThreadPool(1));
+
+    BalancerStrategy balancerStrategy = createBalancerStrategy(exec);
+
+    Map<String, MinMaxPriorityQueue<ServerHolder>> cluster = new HashMap<>();
+    cluster.put("normal", MinMaxPriorityQueue.orderedBy(DruidCoordinatorBalancerTester.percentUsedComparator)
```

Comment:
(Optional) put the orderedBy on a new line and this should auto format much nicer